portfolio/src/main/webapp/leave-comment/script.js

```diff
@@ -1,3 +1,35 @@
+function onSignIn(googleUser) {
+  if ($('#sign-out').length != 0) {
+    return;
+  }
+
+  let loggedUser = googleUser.getAuthResponse().id_token;
+  $('<input />').attr('type', 'hidden')
+    .attr('name', 'userTokenId')
+    .attr('value', loggedUser)
+    .appendTo('#leave-comment-form');
```

Comment:
Do you need to create and remove this field when the user logs in and out? Why not simply set and unset a value of a field which exists all the time?

Reply:
Done.